src/discord/application_command_definitions.rs

```diff
@@ -37,7 +37,7 @@ pub fn application_command_definitions() -> Vec<Command> {
         }
     }
 
-    let create_race = CommandBuilder::new(
+    let create_async_race = CommandBuilder::new(
```

Comment:
when the bot was created, asyncs were the only kind of races, but now they're of very marginal importance. unfortunately a lot of async-related things still just say they're about _races_, so i've been renaming them for clarity just as i run across them and get confused